config/rollup/plugins/styles.js

```diff
@@ -163,8 +159,9 @@ async function generateSass(inputFolder, outputFolder, cssByFile) {
   // We need to transform the contents of the files as some of them contain
   // `:global` css modules definitions that we want to strip out
   const stripGlobalRegex = /:global\s*\(([^)]+)\)|:global\s*{\s*([^}]+)\s*}\s*/g;
+  const globOptions = {cwd: inputFolder, ignore: 'styles/_common.scss'};
```

Comment:
I thought files prefixed with `_` weren’t included by default, is that not the case?

Reply:
Nope all scss files get copied over. I reworked our sass build in #1764 but even before then we always included files starting with an underscore.